app/src/main/java/ar/com/wolox/android/example/ui/home/HomeActivity.kt

```diff
@@ -0,0 +1,24 @@
+package ar.com.wolox.android.example.ui.home
+
+import android.content.Context
+import android.content.Intent
+import ar.com.wolox.android.R
+import ar.com.wolox.wolmo.core.activity.WolmoActivity
+
+class HomeActivity : WolmoActivity() {
+
+    override fun layout(): Int {
+        return R.layout.activity_base
+    }
```

Comment:
When you function is just a return something, you can replace all the `{ return ... }` with just `= ...`. For example:
`override fun layout() = R.layout.activity_base`